app/models/subscriber_list.rb

```diff
@@ -51,4 +55,9 @@ def link_values_are_valid
   def gov_delivery_config
     EmailAlertAPI.config.gov_delivery
   end
+
+  # We could make this more sophisticated if needed, e.g. check document_type.
+  def use_email_alert_frontend_for_email_collection?
+    ENV["USE_EMAIL_ALERT_FRONTEND_FOR_EMAIL_COLLECTION"]
```

Comment:
I would do `ENV.include?` so we always get a boolean here, but I don't think it matters.

Reply:
@thomasleese: updated